docs/checklists/medtronic600.md

```diff
@@ -0,0 +1,236 @@
+  - `[x]` status events (i.e., suspend & resume)
+    - `[ ]` suspensions of insulin delivery are represented as (interval) events with a duration OR
+    - `[x]` suspensions of insulin delivery are represented as pairs of point-in-time events: a suspension and a resumption
+    - `[x]` reason/agent of suspension (`automatic` or `manual`)
+    - `[x]` reason/agent of resumption (`automatic` or `manual`)
+  - calibrations: see [the CGM checklist](CGMChecklist.md) instead
+  - `[x]` time changes (presence of which is also in the [BtUTC section](#bootstrapping-to-utc) below)
+    - `[x]` device display time `from` (before change) and `to` (result of change)
+    - `[ ]` agent of change (`automatic` or `manual`)
+    - `[ ]` timezone
+    - `[ ]` reason for change (read from device)
+
+Device-specific? (Add any device-specific notes/additions here.)
+
+#### SMBG
+
+  - `[x]` blood glucose value
+  - `[x]` subType (`linked` or `manual`)
+  - `[x]` units of value (read from device, not hard-coded)
+  - `*[ ]` out-of-range values (LO or HI)
+  - `*[ ]` out-of-range value thresholds (e.g., often 20 for low and 600 for high on BGMs)
+
+No Tidepool data model yet:
+
+  - `[ ]` meal tag (i.e., pre- or post-meal)
+  - `[ ]` other/freeform tags
+  - `[ ]` categorization of value according to BG target(s) from settings
+
+Device-specific? (Add any device-specific notes/additions here.)
+
+- Bayer meters don't send LO/HI values (or calibrations) to the pump
+
+- Records from mg/dL meters are always in mg/dL. If pump is set to mmol/L, the smbg record will be mg/dL and wizard record will be mmol/L.
+- mmol/L meters do not connect to 523/530G. A mmol/L "native" pump (like the Paradigm Veo) is needed to test what happens if mmol/L meter readings are uploaded.
+
+#### Settings
+
+  - `[x]` basal schedules
+    - `[x]` name of basal schedule OR
+    - `[x]` name of settings profile
+    - `[x]` each schedule as a set of objects each with a rate and a start time
+  - `[x]` name of currently active basal schedule
+  - `[x]` units of all blood glucose-related fields (read from device, not hard-coded)
+  - `[x]` units of all carb-related fields (read from device, not hard-coded)
+  - `[x]` carb ratio(s)
+    - `*[ ]` name of settings profile
+    - `[x]` (one or more) set(s) of objects each with a ratio (amount) and a start time
+  - `[x]` insulin sensitivity factor(s)
+    - `*[ ]` name of settings profile
+    - `[x]` (one or more) set(s) of objects each with an amount and a start time
+  - `[x]` blood glucose target(s)
+    - `*[ ]` name of settings profile
+    - `[x]` (one or more) set(s) of objects each with a target and a start time
+    - target shape:
+        - `[x]` shape `{low: 80, high: 120}` OR
+        - `[ ]` shape `{target: 100}` OR
+        - `[ ]` shape `{target: 100, range: 20}` OR
+        - `[ ]` shape `{target: 100, high: 120}`
+
+Settings history:
+
+  - `*[ ]` device stores all changes to settings OR
```

Comment:
You're storing all changes to settings, right?

Reply:
Yep.